data.json

```diff
@@ -170,6 +170,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "DigitalOcean",
+    "difficulty": "medium",
+    "description": "Make four Pull Requests (PRs) between October 1–31 in any time zone, and earn a limited edition T-shirt!",
+    "reference": "https://hacktoberfest.digitalocean.com/",
+    "image": "https://embed-fastly.wistia.com/deliveries/49bd387c40e2c5aada92abdf973bc46d.webp?image_crop_resized=1920x1080",
```

Comment:
We might also try this image: https://hacktoberfest.digitalocean.com/HF-full-logo.svg

Reply:
yeah sure!